app/models/authn_local.rb

```diff
@@ -49,7 +49,7 @@ def run
             conn.puts issue_token(claims)
```

Comment:
these changes could be removed from this PR. they live on in a separate branch should we choose to pursue them.

Reply:
Agreed.  Alan my run into UI issues with this change.